hdmap/hdmap_shacl.ttl

```diff
@@ -349,7 +215,7 @@ hdmap:QuantityShape a sh:NodeShape ;
             sh:maxCount 1 ;
             sh:message "Validation of speedLimit failed!"@en ;
             sh:name "speedLimit"@en ;
-            sh:node hdmap:Range2DShape ;
+            sh:node general:Range2DShape ;
```

Comment:
as of today 2 pm there is no Range2DShape in general. 
Will it be added or should it be reintroduced here (see line 389 in the diff view on the left)

Reply:
add Range2DShape

Reply:
Question: this is a very general datenstruct and can be used in other shacl's. Better then in General?

Reply:
Range2DShape moved to general and used in hdmap. done!